src/Azure.Functions.Cli/Helpers/AzureHelper.cs

```diff
@@ -722,5 +722,27 @@ public static async Task<FunctionsStacks> GetFunctionsStacks(string accessToken,
                 return null;
             }
         }
+
+        public static async Task<FlexFunctionsStacks> GetFlexFunctionsStacks(string accessToken, string managementURL, string runtime, string region)
+        {
+            // API only supports dotnet as runtime. The dotnet-isolated is part of the dotnet for this API.
+            if (runtime.Equals("dotnet-isolated", StringComparison.OrdinalIgnoreCase))
+            {
+                runtime = "dotnet";
+            }
```

Comment:
Why are we changing this from dotnet-isolated to dotnet? The new stacks API only accepts dotnet as the runtime even though we technically only support dotnet-isolated for flex?

Reply:
Stacks API only has donnet stacks. donnet-isolated stacks are part of the "dotnet" stacks.